packages/validator/src/services/attestation.ts

```diff
@@ -122,7 +166,10 @@ export class AttestationService {
 
     await Promise.all(
       duties.map(async ({duty}) => {
+        // TODO:  duty <> attestation consistency check required? slot and index are checked by lighthouse
```

Comment:
to be removed, really not sure if this is necessary, can this ever happen? @dapplion

Reply:
Consistency is checked in the ValidatorStore